core/mixins/thumbnail.js

```diff
@@ -3,13 +3,20 @@ import { getThumbnailPath as _thumbnailHelper } from '@vue-storefront/core/helpe
 export const thumbnail = {
   methods: {
     /**
-     * Return thumbnail URL for specific base url
-     * @param {String} relativeUrl
-     * @param {Int} width
-     * @param {Int} height
+     * Return thumbnail URL for specific base url and path
+     * @param {string} relativeUrl
+     * @param {number} width
+     * @param {number} height
+     * @param {string} pathType
      */
-    getThumbnail (relativeUrl, width, height) {
-      return _thumbnailHelper(relativeUrl, width, height)
-    }
+    getThumbnail: (relativeUrl, width, height, pathType) => _thumbnailHelper(relativeUrl, width, height, pathType),
```

Comment:
we should change `as` in import and use original helper name
```suggestion
    getThumbnail: (relativeUrl, width, height, pathType) => getThumbnailPath(relativeUrl, width, height, pathType),
```